maps/src/main/java/org/odk/collect/maps/LineDescription.kt

```diff
@@ -0,0 +1,30 @@
+package org.odk.collect.maps
+
+import org.odk.collect.androidshared.utils.toColorInt
+
+data class LineDescription(
+    val points: List<MapPoint>,
+    private val strokeWidth: String?,
+    private val strokeColor: String?,
+    val draggable: Boolean,
+    val closed: Boolean
+) {
+    fun getStrokeWidth(): Float {
+        return try {
+            strokeWidth?.toFloat()?.let {
+                if (it >= 0) {
+                    it
+                } else {
+                    MapConsts.DEFAULT_STROKE_WIDTH
+                }
+            } ?: MapConsts.DEFAULT_STROKE_WIDTH
+        } catch (e: Throwable) {
+            MapConsts.DEFAULT_STROKE_WIDTH
+        }
+    }
+
+    fun getStrokeColor(): Int {
+        val customColor = strokeColor?.toColorInt()
+        return customColor ?: MapConsts.DEFAULT_STROKE_COLOR.toColorInt()!!
```

Comment:
I think we could just store `DEFAULT_STROKE_COLOR` as an `Int` so we don't need to translate it or use `!!`.

Reply:
Nice idea. Done.